packages/forms/resources/views/components/file-upload.blade.php

```diff
@@ -227,9 +227,9 @@ class="w-full"
                                                             <x-filament::button
                                                                 :x-tooltip="
                                                                     '{
-                                                                                                                                        content: ' . \Illuminate\Support\Js::from($action['label']) . ',
-                                                                                                                                        theme: $store.theme,
-                                                                                                                                    }'
+                                                                                                                                                                                                            content: ' . \Illuminate\Support\Js::from($action['label']) . ',
+                                                                                                                                                                                                            theme: $store.theme,
+                                                                                                                                                                                                        }'
                                                                 "
```

Comment:
Could you please undo these (accidental? 😄) formatting changes?

Reply:
I only edited the documentation file using Github UI. I've tried to fix those but 'fix code style' workflow has made these changes again. Maybe it needs a `prettier-ignore`?